pinot-connectors/prestodb-pinot-dependencies/pinot-java-client-jdk8/pom.xml

```diff
@@ -59,6 +59,11 @@
       <artifactId>pinot-spi-jdk8</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.pinot</groupId>
+      <artifactId>pinot-common</artifactId>
```

Comment:
actually, this should be `pinot-common-jdk8`

Reply:
oops, let me sent out another PR once I test that the build works with this. thanks for the catch!

Reply:
@xiangfu0 I've opened this PR to address this: https://github.com/apache/pinot/pull/9353 
can you take a look when you get a chance?